fighthealthinsurance/common_view_logic.py

```
@@ -1562,33 +1602,64 @@ async def generate_appeals(cls, parameters) -> AsyncIterator[str]:
+                logger.debug(
+                    f"Generated {len(ml_citations_context) if ml_citations_context else 0} ML citations for appeal"
+                )
+            except Exception as e:
+                logger.opt(exception=True).debug(
+                    f"Error {e} generating ML citations for {denial}."
+                )
+                ml_citations_context = None
+
         async def save_appeal(appeal_text: str) -> dict[str, str]:
             # Save all of the proposed appeals, so we can use RL later.
             t = time.time()
```

Ruff (0.8.2)

`time` may be undefined, or defined from star imports

(F405)